BambooTracker/main.cpp

```diff
@@ -56,15 +56,16 @@ int main(int argc, char* argv[])
 		QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
 #endif
 
-		std::unique_ptr<QApplicationWrapper> a(std::make_unique<QApplicationWrapper>(argc, argv));
+		QApplication::setHighDpiScaleFactorRoundingPolicy(Qt::HighDpiScaleFactorRoundingPolicy::RoundPreferFloor);
```

Comment:
> This function was introduced in Qt 5.14.
> This enum was introduced or modified in Qt 5.14.

Please guard this line with a `QT_VERSION_CHECK`, check `Qt::AA_EnableHighDpiScaling` just afew lines higher for an example.